lib/instrumentation/span.js

```diff
@@ -153,7 +154,8 @@ Span.prototype._encode = function (cb) {
       duration: self.duration(),
       context: undefined,
       stacktrace: frames,
-      sync: self.sync
+      sync: self.sync,
+      outcome: self.outcome
```

Comment:
@jahtalab What is the `sync` property here for?  Is this related to the `outcome` functionality?

Reply:
The `sync` property is not related to outcome in any way. The change here only adds a comma `,`, is that what you meant?

Reply:
Ah -- I had misread the diff.  We're all good here.